### ISSUE TITLE
Game crashing

### ISSUE THREAD

**Issue by nuker301** [xoxor4d/p2-rtx#117]
When I try to load a or create a new save, the game randomly crashes and doesn't give an error.

Specs:
32gb DDR5 RAM
Ryzen 7 7700x
RX 550
2tb NVME

**Comment by xoxor4d** [comment]
Does it also crash when you start a new game or directly load into a map via console command?

**Comment by nuker301** [comment]
Yes. It loads the loading screen and immediately crashes when, I assume, it starts loading the save.

**Comment by xoxor4d** [comment]
Can you post the logs from `rtx-remix/logs` ?

**Comment by nuker301** [comment]
Where can I find these logs?

**Comment by Powered99** [comment]
Hi, your game crashes because you have an incompatible GPU, a amd rx 550.
You'll unfortunately need an RTX-Series GPU from Nvidia to be able to run RTX remixes, as they use features, like hardware-accelerated ray-tracing, dlss, framegen, etc.
If you'd like to upgrade you can find some used RTX3070's for a good price. Please watch out when buying used GPUs and do your research beforehand tho, because the used gpu market is often filled with scams and ex-bitcoin-mining cards.
For the powerful ryzen 7 7700x, your gpu-cpu combo is pretty unbalanced and such, the rx 550 could really use an upgrade overall, not only in this context 👍

**Comment by nuker301** [comment]
Alright, thank you so much.

**Comment by Powered99** [comment]
Sure thing! If you still want to enhance your visuals before upgrading (or if u cant upgrade), i'd recommend trying out reshade. While it may not add raytracing, it can still make your game look good in some way or another 👍

**Comment by ohuc** [comment]
will it work with a rx 9070xt?

**Comment by Powered99** [comment]
Hi there! Sorry if im late with the reply, but im not a staff member and didnt notice the notification XD. Unfortunately i dont think it will work. The rx 9070xt may be a very powerful gpu, but its an AMD card. As far as i know, only Nvidia RTX gpus are supported by the RTX-remix. Its worth giving a shot tho 👍 !
